src/main/java/com/mixpanel/android/mpmetrics/MixpanelPushNotification.java

```diff
@@ -153,25 +189,39 @@ protected void maybeSetChannel() {
         }
     }
 
+    protected void maybeSetNotificationBadge(NotificationData data) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.BASE && data.badgeCount > 0) {
```

Comment:
just to repeat the question I had in the test code, what happens if you set badgeCount to 0?

Reply:
Doesn't change behavior at all so I though I might as well just not call it